homeassistant/components/accuweather/strings.json

```diff
@@ -41,80 +35,761 @@
+              "moderate": "[%key:component::accuweather::entity::sensor::air_quality_0d::state::moderate%]",
+              "unhealthy": "[%key:component::accuweather::entity::sensor::air_quality_0d::state::unhealthy%]"
             }
           }
         }
+      },
+      "uv_index_4d": {
+        "name": "UV index day 4",
+        "state_attributes": {
+          "level": {
+            "name": "[%key:component::accuweather::entity::sensor::grass_pollen_0d::state_attributes::level::name%]",
+            "state": {
+              "good": "[%key:component::accuweather::entity::sensor::air_quality_0d::state::good%]",
+              "hazardous": "[%key:component::accuweather::entity::sensor::air_quality_0d::state::hazardous%]",
+              "high": "[%key:component::accuweather::entity::sensor::air_quality_0d::state::high%]",
+              "low": "[%key:component::accuweather::entity::sensor::air_quality_0d::state::low%]",
+              "moderate": "[%key:component::accuweather::entity::sensor::air_quality_0d::state::moderate%]",
+              "unhealthy": "[%key:component::accuweather::entity::sensor::air_quality_0d::state::unhealthy%]"
+            }
+          }
+        }
+      },
+      "wet_bulb_temperature": {
+        "name": "Wet bulb temperature"
+      },
+      "wind_speed": {
+        "name": "[%key:component::weather::entity_component::_::state_attributes::wind_speed::name%]"
+      },
+      "wind_chill_temperature": {
+        "name": "Wind chill temperature"
+      },
+      "wind_gust_speed": {
+        "name": "[%key:component::weather::entity_component::_::state_attributes::wind_gust_speed::name%]"
+      },
+      "wind_gust_speed_day_0d": {
+        "name": "Wind gust speed today"
+      },
+      "wind_gust_speed_day_1d": {
+        "name": "Wind gust speed day 1"
+      },
+      "wind_gust_speed_day_2d": {
+        "name": "Wind gust speed day 2"
+      },
+      "wind_gust_speed_day_3d": {
+        "name": "Wind gust speed day 3"
+      },
+      "wind_gust_speed_day_4d": {
+        "name": "Wind gust speed day 4"
+      },
+      "wind_gust_speed_night_0d": {
+        "name": "Wind gust speed night 0"
+      },
+      "wind_gust_speed_night_1d": {
+        "name": "Wind gust speed night 1"
+      },
+      "wind_gust_speed_night_2d": {
+        "name": "Wind gust speed night 2"
+      },
+      "wind_gust_speed_night_3d": {
+        "name": "Wind gust speed night 3"
+      },
+      "wind_gust_speed_night_4d": {
+        "name": "Wind gust speed night 4"
+      },
+      "wind_speed_day_0d": {
+        "name": "Wind speed today"
+      },
+      "wind_speed_day_1d": {
+        "name": "Wind speed day 1"
+      },
+      "wind_speed_day_2d": {
+        "name": "Wind speed day 2"
+      },
+      "wind_speed_day_3d": {
+        "name": "Wind speed day 3"
+      },
+      "wind_speed_day_4d": {
+        "name": "Wind speed day 4"
+      },
+      "wind_speed_night_0d": {
+        "name": "Wind speed night 0"
```

Comment:
```suggestion
        "name": "Wind speed tonight"
```

Reply:
Or does this value target previous night or next night?

Reply:
It's a forecast, so it's about the night that will come. I think `tonight` is more appropriate.